users/models.py

```diff
@@ -13,10 +14,21 @@ def validate_email_addr(value):
 
 
 def validate_phone_number(value):
```

Comment:
can we use some other variable name?
e.g

def validate_phone_number(**phone_number**):

Reply:
Done